index.js

```diff
@@ -506,8 +507,12 @@ HyperDB.prototype.list = function (prefix, opts, cb) {
   }
 }
 
-HyperDB.prototype.history = function (opts) {
-  return history(this, opts)
+HyperDB.prototype.history = function (prefix, opts) {
+  if (prefix && !isOptions(prefix)) {
+    console.log('KEY')
```

Comment:
prob didn't mean to add this?

Reply:
Ah no I totally did not - good catch. Thanks!